src/components/FlowRunCreateFormV2.vue

```diff
@@ -158,6 +168,7 @@
       },
       name: name.value,
       parameters: parameters.value,
+      jobVariables: jobVariables.value ? JSON.parse(jobVariables.value) : undefined,
```

Comment:
Should this just be part of the mapper?

Reply:
It could be. 

tho it might be better here as it being a string is more of an implementation detail of the form that it's in so that it plops into a textarea. maybe if `DeploymentFlowRunCreate` is actually `FlowRunCreateFormValues` and so then a mapper going from `FlowRunCreateFormValues` -> `DeploymentFlowRunCreateRequest` seems sensible to encapsulate the stringifying. 

that gel w/ ur mental model too?

Reply:
Both seem logical. Being an implementation detail makes sense too, probably wouldn't of thought about the mapper if I hadn't seen it in both form components. But one of these will go away in a week or so anyway.